Code/GraphMol/SmilesParse/smiles.yy

```diff
@@ -35,27 +37,56 @@ namespace {
   molList->resize(0);
  }
 }
+
+void printSyntaxErrorMessage(std::string_view input,
+                             std::string_view err_message,
+                             unsigned int bad_token_position) {
+    // NOTE: If the input is very long, the pointer to the failed location
+    // becomes less useful. We should truncate the length of the error message
+    // to 101 chars.
+    static constexpr unsigned int error_size{101};
```

Comment:
Why 101? I take it is an arbitrary value, so it might be good to mention that in the comment.

Also, since we are printing the whole SMILES before the chunk where the error is located, I think we might to go something even smaller (41 chars, 20 on each side of the problem?) to no be too redundant.

Reply:
just updated this to `41`